todo/src/index.js

```diff
@@ -0,0 +1,101 @@
+import StorageController from "./scripts/storageController";
+import themeToggler from "./scripts/themeToggler";
+import * as sortBarView from  "./scripts/sortBarView";
+import * as tasksFieldView from "./scripts/tasksFieldView";
+import * as modalForm from "./scripts/modalForm";
+import generateId from "./scripts/idGenerator";
+import setCounter from "./scripts/itemCounter";
+import * as dictionary from "./scripts/dictionary";
+import "./style/index.scss";
+import "./style/toggler.scss";
+
+themeToggler();
+
+class ToDo {
+    constructor(){
+        this.storageController = new StorageController();
+        modalForm.addSubmitHandler(this.modalSubmitCallback.bind(this));
+        tasksFieldView.setTasksContainerClickHandler(this.changeTaskMode.bind(this));
+        this.sortDirection = this.storageController.getSortDirection();
+        sortBarView.initChangeDirectionHandler(this.sortDirection, this.changeTaskSortDirection.bind(this));
+        this.sortDirection = this.storageController.getSortDirection();
+        tasksFieldView.setSortDirection(this.sortDirection);
+        this.refreshTasksField();
+    }
+
+    changeTaskSortDirection(isChange) {
+        if (isChange) {
+            tasksFieldView.setSortDirection(this.storageController.changeSortDirection());
+            this.refreshTasksField();
+        }
+    }
+
+    refreshTasksField() {
+        const tasksData = this.storageController.getItemsList();
+        tasksFieldView.refreshTaskFields(tasksData);
+        setCounter(tasksData[0].length ,tasksData[1].length);
+    }
+
+    changeTaskMode(taskId, modeWord) {
+        const mode = dictionary.modes[modeWord];
+        if (mode === 'delete') {
```

Comment:
why do you not use `modes` in comparisons ?

Reply:
I rewrote this.
```suggestion
    changeTaskMode(taskId, modeWord) {
        if (dictionary.modes[modeWord] === 'delete') {
            this.deleteTask(taskId);
        } 
        else if (dictionary.modes[modeWord] === 'complete' || dictionary.modes[modeWord] === 'uncomplete') {
            this.storageController.changeItemStatus(taskId);
        }
        else if(dictionary.modes[modeWord] === 'edit') {
            this.editTask(taskId)
        }
        this.refreshTasksField();
    }
```